01_GettingStartedWithDataLab/01_JupyterPythonSQL101.ipynb

```diff
@@ -1676,57 +1701,74 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Asynchronous queries get a jobid upon submission, e.g.:\n",
```

Comment:
Please restore the original content of this section on async queries. This is important for the automatic notebook testing  to run through.

Reply:
This section isn't reverted to the original version yet. I can't approve the merge of this PR before...